src/back/controllers/api/index.js

```diff
@@ -3,4 +3,5 @@ const router = require('express').Router;
 
 module.exports = router()
     .use('/challenges', require('./routes/challenges'))
```

Comment:
@borys-ihnatyev, in you're right but can we do the refactoring required here not in the scope of this story?